src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -94,9 +94,37 @@ If you want to customize APM agent behavior, you can use our APM agent APIs. Her
+Use this method if you can't use APM for whatever reason, like if your APM agent isn't supported for auto-injection (as is true for Node.js and some Java apps). If you use this method, you'll receive a customized JavaScript snippet at the end of the install process in the New Relic UI, and you add that to the HTML of any pages you want to monitor in your app/site.
 
-The JavaScript snippet should be pasted into the `<head>` tag of views in your app. The head tag typically handles third party scripts, fonts, and other elements that load on all your app pages, so you'll only need to paste it into one place. Paste it as close to the top of the `<head>` as possible, but after any position-sensitive META tags (such as X-UA-Compatible and charset). For more about the snippet placement, see [JavaScript placement requirements](/docs/browser/new-relic-browser/page-load-timing-resources/instrumentation-browser-monitoring#javascript-placement).
+The JavaScript snippet should be pasted within the HTML `<head>` tag of views in your app. The head tag typically handles third party scripts, fonts, and other elements that load on all your app pages, so you'll only need to paste it into one place. Paste it as close to the top of the `<head>` as possible, but after any position-sensitive META tags (such as X-UA-Compatible and charset). For more about the snippet placement, see [JavaScript placement requirements](/docs/browser/new-relic-browser/page-load-timing-resources/instrumentation-browser-monitoring#javascript-placement).
 
+Alternatively and while we STRONGLY recommend the inline script path, you can also use a src script to insert the agent in an `async` or `defer` manner. For caveats and details, continue below.
 
-<InstallFeedback />
\ No newline at end of file
+### Delaying browser monitoring [#delayed-script]
+
+<Callout variant="caution">
+  Delaying instantiation will cause the agent to miss information until such time that it is fully loaded and initialized. We assume you understand the order of execution for these scripts and the loss in instrumentation data from the start of page views.
+</Callout>
+
+For possible performance or other reasons, you may wish to delay the initialization of the browser agent from the start of the page load cycle. While this can be done, you should be aware of the implications such as inability to use the `newrelic` API until the agent is ready and incomplete data in your New Relic view.
+
+For some features, early page life data will not be captured. For others, it may be extrapolated in best faith. The agent is still expected to function after it has been initialized, whenever that occurs. The tardiness in which the agent will wrap globals or   subscribe to events relative to other libraries or code will affect captured data as well.
```

Comment:
```suggestion
Once the agent is fully initialized, it functions as expected. Any delay also affects when the agent wraps globals or subscribes to events relative to other libraries or code that affects captured data.
```

I deleted the first two sentences because they're vague and not specific enough.